JavaScript/JavaScriptSDK/appInsights.ts

```diff
@@ -126,6 +133,11 @@ module Microsoft.ApplicationInsights {
                 }
 
                 this._pageTracking.stop(name, url, properties, measurements);
+
+                if (this.config.autoTrackPageVisitTime) {
+                    this.autoTrackPageVisitTime(name, url);
```

Comment:
So these lines do this:
1. Track PageView for the current page.
2. Track Page Visit Time for the previous page (since we are on a new page, we know that the visit time of the previous page is over).

We are working on moving the custom metric to a standard page visit time metric, but there is still work to be done in the backend.